lib/commands/deprecated/deprecated-command.js

```diff
@@ -0,0 +1,12 @@
+var chalk       = require('chalk');
+var SilentError = require('ember-cli/lib/errors/silent');
+
+module.exports = function(command) {
+  var description = command.description || '';
+  command.description = chalk.red('[DEPRECATED] ') + description;
+  command.run = function() {
+    throw new SilentError('This command has been deprecated.  Please see `https://github.com/ember-cli/ember-cli-deploy` for details.');
```

Comment:
nit: "deprecated" means still works but will be removed in the future. We should probably refer to this as "no longer supported"